admin/bulk-inactive-transfer.vbs

```diff
@@ -72,109 +77,145 @@ Do
 	CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
 Loop until are_we_passworded_out = false					'loops until user passwords back in
 
+Dialog1 = ""
+BeginDialog Dialog1, 0, 0, 126, 50, "Select the excel row to start"
+  EditBox 75, 5, 40, 15, excel_row_to_restart
+  ButtonGroup ButtonPressed
+    OkButton 10, 25, 50, 15
+    CancelButton 65, 25, 50, 15
+  Text 10, 10, 60, 10, "Excel row to start:"
+EndDialog
+
+DO
+    dialog Dialog1
+    If buttonpressed = 0 then stopscript								'loops until all errors are resolved
+    CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
+LOOP UNTIL are_we_passworded_out = false					'loops until user passwords back in
+
+excel_row = excel_row_to_restart
+
+
+back_to_self
+EMWriteScreen CM_mo, 20, 43
+EMWriteScreen CM_yr, 20, 46
+transmit
+
 ObjExcel.Cells(1, 1).Value = "WORKER"
 ObjExcel.Cells(1, 2).Value = "CASE NUMBER"
 ObjExcel.Cells(1, 3).Value = "CASE NAME"
 ObjExcel.Cells(1, 4).Value = "APPL DATE"
 ObjExcel.Cells(1, 5).Value = "INAC DATE"
 ObjExcel.Cells(1, 6).Value = "TRANSFERED"
 ObjExcel.Cells(1, 7).Value = "CONFRIM"
-
-FOR i = 1 to 7		'formatting the cells'
-	objExcel.Cells(1, i).Font.Bold = True		'bold font'
-	'ObjExcel.columns(i).NumberFormat = "@" 		'formatting as text
-	objExcel.Columns(i).AutoFit()				'sizing the columns'
-NEXT
+STATS_counter = STATS_counter + 1
```

Comment:
Having the stats_counter here is only going to increment the count once. 

Line 121 - trim after exiting the do won't work. Swap lines 120 and 121. Then add the stats_counter incrementor.

Reply:
REMOVED THE LINE AND COMBINED INTO PREVIOUS